yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -68,6 +68,32 @@ module vyatta-protocols-frr-bgp-v1 {
 		description "Inital version.";
 	}
 
+  typedef rt-rd {
```

Comment:
This file uses Tab characters for indentation. Please could you use the same for your additions?

Reply:
We tried to fix this issue. 
We will send for review shortly.